RecoLocalTracker/SiStripClusterizer/interface/SiStripClusterizerConditionsStruct.h

```diff
@@ -0,0 +1,23 @@
+#ifndef RecoLocalTracker_SiStripClusterizer_interface_SiStripClusterizerConditionsStruct_h
+#define RecoLocalTracker_SiStripClusterizer_interface_SiStripClusterizerConditionsStruct_h
+
+#include "DataFormats/SiStripCommon/interface/ConstantsForHardwareSystems.h"
+
+namespace sistrip {
+  // The underscore, such as `qualityOk_`, indicates the array is indexed with the channel index.
```

Comment:
I don't understand the comment - aren't _all_ data members of both structs ending with a trailing `_` ?